### ISSUE TITLE
Неполноценные и непонятные issue(задачи/проблемы)

### ISSUE THREAD

**Issue by Electro98** [Electro98/cyber-growbox-2077#25]
# Неполноценные и непонятные issue(задачи/проблемы)
## Описание проблемы

Наш cyber-growbox-2077 является успешным учебным проектом, но у него есть серьёзный недостаток/преимущество, он затрагивает темы шире наших профессиональных тем из-за чего преподаватели не могут качественно проверить прогресс нашей работы.

Мы им увы не помогаем и issue, написанные нами, недостаточно информативны и почти не описывают проблем/задач, которые они должны решить.

## Предлагаемое решение

Всем студентам, работающим над этим проектом, таким как: @Timofey-arch, @SerLap-ctrl, @Electro98. 

Необходимо научиться:
- Использовать стиль Markdown;
- Внятно описывать проблему при написании issue;
- Использовать теги в виде смайликов в сообщении коммита, в соотвествии с рекомендациями описанными в файле `CONTRIBUTING.md`.

Также предлагается оформить полноценную веху проекта связанную с завершением этапа тестирования отдельного функционала микроконтроллера, датчиков и модулей.

## Система контроля выполнения этой задачи

Предлагается назначить @redb0, как человека контроллируещего процесс обучения необходимым навыкам. А @NikMatyukhin, как человек непосредственно не связанный с проектом, будет объективно оценить результат обучения.

## Дополнительные сведения

Любые предложения и комментарии по поводу данной проблемы будут приняты к рассмотрению.

**Comment by redb0** [comment]
Исчерпывающее руководство по Markdown для github можно прочитать [здесь](https://gist.github.com/Jekins/2bf2d0638163f1294637) на русском языке.

Также необходимо коротко, но ёмко задавать описание коммита и придерживаться правила: одно изменение - один коммит.

**Comment by NikMatyukhin** [comment]
Все наблюдения и объективные оценки разрешается писать в ветку комментариев этой задачи, или оставить только на словах при личной встрече?

**Comment by Electro98** [comment]
> Все наблюдения и объективные оценки разрешается писать в ветку комментариев этой задачи, или оставить только на словах при личной встрече?

Да, конечно, любой формат приветствуется. Можно свободно оставлять тут любые наблюдения.
